### PR TITLE
chore: Create commons module

### PR DESCRIPTION
**Pull request by dsimansk**

### CODE REVIEW

File: rest-tests/src/test/java/io/syndesis/qe/rest/tests/ClientTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@Slf4j
+public class ClientTest {
+
+	@Test
+	public void testDC() throws InterruptedException {
+
+		SyndesisTemplate.deploy();
+
+
```
Comment (tplevko) [inline rest-tests/src/test/java/io/syndesis/qe/rest/tests/ClientTest.java]: tomorrow discussion about checkstyle?

File: utilities/pom.xml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+			<artifactId>jackson-datatype-jsr310</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-slf4j-impl</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>${guava.version}</version>
```
Comment (tplevko) [inline utilities/pom.xml]: version

File: utilities/pom.xml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>json-path</artifactId>
+			<type>jar</type>
+		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>json-schema-validator</artifactId>
+			<version>${rest.assured.version}</version>
```
Comment (tplevko) [inline utilities/pom.xml]: version

File: rest-tests/src/test/java/io/syndesis/qe/rest/tests/ClientTest.java (hunk 1 above)
Comment (dsimansk) [reply rest-tests/src/test/java/io/syndesis/qe/rest/tests/ClientTest.java]: This is my to-be-removed test class :)

File: pom.xml [hunk 4]
```diff
@@ -40,8 +59,194 @@
...
+			<modules>
+				<module>rest-tests</module>
+				<module>ui-tests</module>
+				<module>utilities</module>
+			</modules>
+		</profile>
+		<profile>
 			<id>rest</id>
 			<activation>
 				<property>
```
[the remark this answers is not in the document]
Comment (dsimansk) [reply pom.xml]: Actually this should be fine until `utilities` is defined in default modules.

[the remark this answers is not in the document]
Comment (tplevko) [reply pom.xml]: that's true

**Pull request merged**